e2e-bfd-test/src/test/resources/application.properties

```diff
@@ -90,4 +89,13 @@ bfd.keystore.location=${AB2D_BFD_KEYSTORE_LOCATION}
 
 audit.files.ttl.hours=72
 
-eob.job.patient.queue.page.size=1000
\ No newline at end of file
+eob.job.patient.queue.page.size=1000
+# Number of EOBs to search per file
+eob.job.patient.number.per.thread=10
+
+aggregator.multiplier=3
+aggregator.directory.finished=finished
+aggregator.directory.streaming=streaming
+aggregator.file.buffer.size=0
+
+eob.job.patient.number.per.thread=10
```

Comment:
Can you document why each of these is being set the way it is?

Do these variables even need to be set if they're provided in the worker properties file already?

Reply:
Removed them especially since I now pass in an object.